Formula/xplanet.rb

```diff
@@ -38,6 +38,11 @@ class Xplanet < Formula
   depends_on :x11 => :optional
 
   def install
+    # This is a simple fix to a bug in Xplanet 1.3.1 (-num_times=2 doesn't work):
+    # http://xplanet.sourceforge.net/FUDforum2/index.php?t=msg&th=671&start=0&S=07920715eb9a74361428ce957b99ae66
+    # This block should be removed with the next release of Xplanet
+    inreplace "src/libdisplay/DisplayOutput.cpp", "if (stopIndex > 1)", "if (stopIndex > 0)"
```

Comment:
Sure!

In this case, you'll need to submit a PR to [formula-patches](https://github.com/Homebrew/formula-patches) with the patch in the URL above - name it something like `xplanet/xplanet-1.3.1-ntimes.patch`. 

Then, in this formula, add a `patch` block that looks like this (example comes from `cctools.rb`):

```ruby
patch :p0 do
    url "https://raw.githubusercontent.com/Homebrew/formula-patches/db27850/cctools/cctools-829-lto.patch"
    sha256 "8ed90e0eef2a3afc810b375f9d3873d1376e16b17f603466508793647939a868"
end
```

(Replace the `url` above with the patch you submitted).